Documentation/Books/Manual/Programs/Arangorestore/Examples.md

```diff
@@ -142,44 +151,56 @@ you can start with the following command:
 
     arangodump --collection myvalues --server.database mydb --output-directory "dump"
 
-This will create two files, *myvalues.structure.json* and *myvalues.data.json*, in the output 
-directory. To load data from the datafile into an existing collection *mycopyvalues* in database 
+This will create two files, *myvalues.structure.json* and *myvalues.data.json*, in the output
+directory. To load data from the datafile into an existing collection *mycopyvalues* in database
 *mycopy*, rename the files to *mycopyvalues.structure.json* and *mycopyvalues.data.json*.
 After that, run the following command:
-    
-    arangorestore --collection mycopyvalues --server.database mycopy --input-directory "dump"
-
-Using arangorestore with sharding
----------------------------------
 
-As of Version 2.1 the *arangorestore* tool supports sharding. Simply
-point it to one of the coordinators in your cluster and it will
-work as usual but on sharded collections in the cluster.
+    arangorestore --collection mycopyvalues --server.database mycopy --input-directory "dump"
 
-If *arangorestore* is asked to restore a collection, it will use the same number of 
-shards, replication factor and shard keys as when the collection was dumped. 
-The distribution of the shards to the servers will also be the same as at the time of the dump,
-provided that the number of DBServers in the cluster dumped from is identical to the
-number of DBServers in the to-be-restored-to cluster.
+Restoring in a Cluster
+----------------------
 
-To modify the number of _shards_ or the _replication factor_ for all or just some collections,
-*arangorestore*, starting from v3.3.22 and v3.4.2,  provides the options `--number-of-shards` and `--replication-factor`.
-These options can be specified multiple times as well, in order to override the settings
+From v2.1 on, the *arangorestore* tool supports sharding and can be
+used to restore data into a Cluster. Simply point it to one of the
+_Coordinators_ in your Cluster and it will work as usual but on sharded
+collections in the Cluster.
+
+If *arangorestore* is asked to drop and re-create a collection, it
```

Comment:
this paragraph was removed in https://github.com/arangodb/arangodb/commit/b3a3e35f3dfaf06ee6e896695ddc051857c5aa52#diff-57875d30de31f16d6048c4df9cba987d - removing it from here as well (part of conflict resolution)